framework/BaseForm.js

```diff
@@ -0,0 +1,19 @@
+import { Timeouts } from '../constants/timeouts.js'
+
+class BaseForm {
+  constructor(name, selector) {
+    this.name = name
+    this.selector = selector
```

Comment:
here you can have new Label(this.locator, this.name) for instance and use it in isDisplayed method